Moda.Web/src/moda.web.reactclient/app/services/moda-api.ts

```diff
@@ -185,12 +185,7 @@ export class ProfileClient {
             }
         }
         if (status === 200) {
-            const _responseText = response.data;
-            let result200: any = null;
-            let resultData200  = _responseText;
-            result200 = JSON.parse(resultData200);
-            return Promise.resolve<string[]>(result200);
-
+            return Promise.resolve<string[]>(response.data);
```

Comment:
response.data is an array, not a string, so this generated client threw an exception on JSON.parse.
I would be surprised if we don't run into that issue in the future.